samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -489,6 +506,12 @@ function createBatchedWithVertexColors() {
     return saveBatchedTileset('BatchedWithVertexColors', tileOptions);
 }
 
+function createBatchedWithDataURI() {
+    return saveBatchedTileset('BatchedWithContentUri', undefined, {
```

Comment:
Should the tileset name be `BatchedWithContentDataUri`? Just so it's clear it's a data uri.